microsoft/testsuites/display/modetest.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix, Redhat, Ubuntu
+from lisa.tools.git import Git
+from lisa.util import LisaException
+
+
+class Modetest(Tool):
+    repo = "https://github.com/grate-driver/libdrm"
```

Comment:
Let's stick to a long term supported branch to prevent breaking changes.

Reply:
I didn't see such branches, and no tags, the latest change for this rero happened on 4 years ago.